cmd/fleet-manager/scheme/scheme.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package scheme
 
 import (
+	hrapiv2b1 "github.com/fluxcd/helm-controller/api/v2beta1"
+	sourcev1 "github.com/fluxcd/source-controller/api/v1beta2"
```

Comment:
why not use github.com/fluxcd/source-controller/api/v1

Which schema do you use

Reply:
`v0.32.0`, I should upgrade api to `1.0.0-rc1`,

Reply:
v1 is unavailable in helm chart: https://github.com/fluxcd-community/helm-charts/blob/main/charts/flux2/templates/source-controller.crds.yaml